src/_help/customization-options/logo-meta-images.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Logo and Meta images
+---
+
+- TOC
+{:toc}
+
+## Logo and favicon
+
+You can upload your own logo and favicon on Bump.sh to customize your documentation with your brand or project colors.
+
+From the "Customize UI" tab in the documentation settings, you can add your own logo and favicon.
+
+![](/images/help/logo-favicon.png)
+
+To provide the best image quality, it is recommended to adhere to the dimensions below:
+- Logo:
+- Favicon:
+
+## Meta images
+
+A "meta image" typically refers to an image that is used as a preview image when a web page is shared on social media platforms and can be displayed in search engine results. You may have already heard it referred to as a "social media image." Meta images help make the links posted on social media platforms more attractive, especially when shared or displayed in posts.
+
+### Add your own image
+
+From the "Customize UI" tab in the settings of a documentation, you can add your own meta image.
+
+It is advisable not to exceed a width of 1200px and to maintain the height between 600px and 700px for better integration on most platforms.
+
+![](/images/help/meta-image.png)
+
+### Use the automated image generation
+
+In case you wouldn't have a visual you could use, Bump will automatically generate a meta image. It displays your documentation name, its logo and its type.
+
+![](/images/help/meta-image-example.png)
+
+> The social media image will be modified each time you change the name or the logo.
+{: .info}
+
+> First generation of this new social media image may take a bit, but any modification or change will start the generation process.
+{: .warning}
+
+## Hub customization
+
+All of these customization settings can be applied to all the documentations within a hub.
```

Comment:
```suggestion
All of these customization settings can be applied to all documents within a documentation hub.
```

Reply:
Not sure of the usage here of "documents"

Reply:
But fixed the typo